disttask/importinto/dispatcher.go

```diff
@@ -584,8 +598,6 @@ func rollback(ctx context.Context, handle dispatcher.TaskHandle, gTask *proto.Ta
 
 func stepStr(step int64) string {
 	switch step {
-	case proto.StepInit:
```

Comment:
Does this code related with dynamic generate subtasks？

Reply:
> Does this code related with dynamic generate subtasks？

Yes, dynamic generate subtasks changes some of task.step updating process.

Reply:
> > Does this code related with dynamic generate subtasks？
> 
> Yes, dynamic generate subtasks changes the task.step updating process. The framework manage the step updating instead of DispatcherExt